examples/rust/external_data_loader/src/main.rs

```diff
@@ -21,14 +24,30 @@ use rerun::{MediaType, EXTERNAL_DATA_LOADER_INCOMPATIBLE_EXIT_CODE};
 /// Rust source file with Rerun (`rerun file.rs`).
 ///
 /// [`rerun-loader-`]: `rerun::EXTERNAL_DATA_LOADER_PREFIX`
-#[derive(argh::FromArgs)]
+#[derive(argh::FromArgs, Debug)]
 struct Args {
     #[argh(positional)]
     filepath: std::path::PathBuf,
 
-    /// optional ID of the shared recording
+    /// optional recommended ID for the recording
     #[argh(option)]
     recording_id: Option<String>,
+
+    /// optional prefix for all entity paths
+    #[argh(option)]
+    entity_path_prefix: Option<String>,
+
+    /// optional mark data to be logged as timeless
+    #[argh(switch)]
+    timeless: bool,
+
+    /// optional timestamps to log at (e.g. `--time sim_time=1709203426`) (repeatable)
+    #[argh(option)]
+    time: Vec<String>,
+
+    /// optional sequences to log at (e.g. `--sequence sim_frame=42`) (repeatable)
+    #[argh(option)]
+    sequence: Vec<String>,
```

Comment:
wondering if there's a way for data loader authors to "inherit" these arguments. Can clap work with nested arg strucs..?